docs/source/methods/depletion.rst

```diff
@@ -257,3 +257,68 @@ choose one of two methods for estimating the heating rate, including:
 
 The method for normalization can be chosen through the ``normalization_mode``
 argument to the :class:`openmc.deplete.CoupledOperator` class.
+
+----------------
+MSR Capabilities
+----------------
+
+A nice feature of Molten Salt Reactors (MSRs), and in general of
+*any*  liquid fuel reactor, is the capability to continuously reprocess the fuel
+to remove fission products or to feed fresh fuel into the system.
+
+Removal rate
+------------
+
+A removal rate as defined here is the rate at which nuclides are
+continuously removed from a material. T Mathematically, it
+can be thought as an additional proportional term :math:`\mathbf{T}`
+to the depletion equation, that can be rearranged as:
+
+.. math::
+
+  \begin{aligned} \frac{dN_i(t)}{dt} = \underbrace{\sum_j \int_0^\infty dE;
+  \f_{j\rightarrow i} N_j(t) \sigma_j (E,t) \phi(E,t)} - N_i(t) \sigma_i(E,t) \phi(E,t)}_\textbf{R}  +
+  \underbrace{\sum_j \lambda_{i\rightarrow j}  N_j(t) + \lambda_{j\rightarrow i}  N_i(t)}_\textbf{D} -
+  \underbrace{\epsilon_i \lambda_i N_i(t)}_\textbf{T} \end{aligned}
+
+where $\epsilon_i$ is the removal rate of nuclide $i$. Here the transmutation terms :math:`\mathbf{R}` and the decay terms :math:`\mathbf{D}`
```

Comment:
```suggestion
  \begin{aligned}\frac{dN_i(t)}{dt} = &\underbrace{\sum\limits_j f_{j\rightarrow i}
  \int_0^\infty dE  \; \sigma_j (E,t) \phi(E,t) N_j(t)  - \int_0^\infty dE \; \sigma_i(E,t) 
  \phi(E,t) N_i(t)}_\textbf{R} \\ 
  &+ \underbrace{\sum_j \lambda_{j\rightarrow i} N_j(t) + \lambda_{i\rightarrow j} N_i(t)}_\textbf{D} \\
  &- \underbrace{\epsilon_i \lambda_i N_i(t)}_\textbf{T} \end{aligned}

where :math:`\epsilon_i` is the removal rate of nuclide :math:`i`. Here the transmutation terms :math:`\mathbf{R}` and the decay terms :math:`\mathbf{D}`
```
Looks like I led you a bit astray with my last suggestion. This should fix the the equation rendering.